Neos.Fusion/Classes/FusionObjects/AbstractArrayFusionObject.php

```diff
@@ -50,6 +50,22 @@ public function setIgnoreProperties($ignoreProperties = [])
         $this->ignoreProperties = $ignoreProperties;
     }
 
+    /**
+     * Checks wether this Array fusion object should have sorted properties (according to __meta.position) or not
+     *
+     * @return bool
+     * @see applyPositionalArraySorterToProperties
+     */
+    public function shouldSortProperties(): bool
```

Comment:
mhh why is this `public`?

Reply:
Interesting question, I followed what we do elsewhere in fusion implementations with thse kind of accessors, but I guess we could indeed make it protected at least

Reply:
I'm fine with leaving it then, you're right we do that always - weird code style but whatever

Reply:
Something to question and fix everywhere I guess. I'll see if I can figure out why we started it...

Reply:
Comment was unclear, but I meant figure out in a separate PR...